src/lib/Libnet/net_server.c

```diff
@@ -461,24 +443,20 @@ engage_authentication(conn_t *pconn)
 
 	if (ret == CS_AUTH_CHECK_PORT) {
 		/*dealing with STD security's  "equivalent of"  CS_sever_auth*/
-
 		if (pconn->cn_authen & PBS_NET_CONN_FROM_PRIVIL)
 			pconn->cn_authen |= PBS_NET_CONN_AUTHENTICATED;
 		return (0);
 	}
 
 	(void)get_connecthost(sd, ebuf, sizeof(ebuf));
 
-	sprintf(logbuf,
-		"Unable to authenticate connection from (%s:%d)",
+	sprintf(logbuf,	"unable to authenticate connection from (%s:%d)",
```

Comment:
Why change the message text?

Reply:
I saw most of our error messages (atleast in net_server.c) did not start with Capitals and did not end with a period. Hence to make it similar. However, if you prefer, I can change the log text's back.

Reply:
If the tests pass, I'm fine with the change. I'm simply concerned about regressions. These are the kind of "common sense" changes that should not require an EDD, IMHO.

Reply:
Thanks!